cwms-data-api/src/test/resources/cwms/cda/data/dto/project.json

```diff
@@ -13,10 +13,14 @@
   "sedimentation-desc" : "Sedimentation Description",
   "downstream-urban-desc" : "Downstream Urban Description",
   "bank-full-capacity-desc" : "Bank Full Capacity Description",
-  "pump-back-location-id" : "Pumpback Location Id",
-  "pump-back-office-id" : "SPK",
-  "near-gage-location-id" : "Near Gage Location Id",
-  "near-gage-office-id" : "SPK",
+  "pump-back" : {
```

Comment:
I feel like this should still say "-location", but that might be just me. @adamkorynta @DanielTOsborne @krowvin 

(Don't wait forever for comment, just tagging for possible thoughts now.)

Reply:
I agree with `pump-back-location` to be absolutely clear it isn't operational data

Reply:
Adding -location Makes sense to me.